assets/olive-optimizer/general/jobs/olive-optimizer-built-in-component/pipeline.yml

```diff
@@ -0,0 +1,20 @@
+$schema: https://azuremlschemas.azureedge.net/latest/pipelineJob.schema.json
+type: pipeline
+display_name: olive-optimizer-built-in-component
+description: Use a built-in olive_optimizer component to optimize a bert model.
+
+compute: azureml:cpu-cluster
+
+jobs:
+  optimize:
+    type: command
+    component: file:../../components/cpu/spec.yaml
```

Comment:
Please reference a component by name from the registry, not using the `file` schema. You can use azureml:<compoent_name>:<component_version> syntax.